tests/run/ext_attr_getter.srctree

```diff
@@ -78,32 +111,89 @@ cdef extern from "foo_nominal.h":
             int field2 "f2"
 
 def sum(Foo f):
-    # the f.__getattr__('field0') is replaced in c by f->f0
+    # Note - not a cdef function but compiling the f.__getattr__('field0')
+    # notices the alias and replaces the __getattr__ in c by f->f0 anyway
     return f.field0 + f.field1 + f.field2
 
+######## getter1.pyx ########
+
+# Access base Foo fields from C via getter functions
+
+
+cdef extern from "foo.h":
+    ctypedef class foo_extension.Foo [object FooStructOpaque, check_size ignore]:
+        @property
+        cdef int field0(self):
+            return PyFoo_GET0M(self)
+
+        @property
+        cdef int field1(self):
+            return PyFoo_Get1F(self)
+
+        @property
+        cdef int field2(self):
+            return PyFoo_GET2M(self)
+
+    int PyFoo_GET0M(Foo);  # this is actually a macro !
+    int PyFoo_Get1F(Foo);
+    int PyFoo_GET2M(Foo);  # this is actually a macro !
+
+def sum(Foo f):
+    # Note - not a cdef function but compiling the f.__getattr__('field0')
+    # notices the getter and replaces the __getattr__ in c by PyFoo_GET anyway
+    return f.field0 + f.field1 + f.field2
```

Comment:
We should test that the C level access is used here and not the Python level access. Not sure if [tree assertions](https://github.com/cython/cython/wiki/HackerGuide#parse-tree-assertions) work in srctree tests (they might not), but they seem necessary here, give that the Python interface is the same.

Reply:
I changed the attribute names in both the ctypedef class definition and `sum()`.